ivy/functional/ivy/elementwise.py

```diff
@@ -4179,7 +4179,7 @@ def log1p(
 @handle_array_function
 @handle_device_shifting
 def log2(
-    x: Union[ivy.Array, ivy.NativeArray],
+    x: Union[ivy.Array, ivy.NativeArray, Number],
```

Comment:
Any reason why Number was added as supported type?

Reply:
Sorry for this  mistake ,I will fix it

Reply:
@Dsantra92  i fix it now ..can you check it again?